plugins/gcp/resoto_plugin_gcp/__init__.py

```diff
@@ -118,7 +123,7 @@ def collect_project(
 
         try:
             core_feedback.progress_done(project_id, 0, 1)
-            gpc = GCPProjectCollector(project)
+            gpc = GcpProjectCollector(Config.gcp, Cloud(id="Gcp", name="Gcp"), project, core_feedback)
```

Comment:
Every collector has `self.cloud` which contains an instance of `Cloud()`.

Reply:
@lloesche Did you mean "every colelctor plugin" 🤔 ? Because this is where the project collector is getting their "self".cloud.